modules/core/src/controllers/orthographic-controller.js

```diff
@@ -107,11 +107,16 @@ class OrthographicState extends ViewState {
    * @param {Number} scale - a number between [0, 1] specifying the accumulated
    *   relative scale.
    */
-  zoom({scale}) {
-    const {zoom} = this._viewportProps;
+  zoom({pos, scale}) {
```

Comment:
Nit: update the description, zoom with the current mouse pointer as center? assuming that is what this method is doing.